src/measure_theory/measure/complex.lean

```diff
@@ -0,0 +1,132 @@
+/-
+Copyright (c) 2021 Kexing Ying. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Kexing Ying
+-/
+
+import measure_theory.decomposition.jordan
+
+/-!
+# Complex measure
+
+This file proves some elementary results about complex measures. In particular, we prove that
+a complex measure is always in the form `s + it` where `s` and `t` are signed measures.
+
+The complex measure is defined to be vector measure over `ℂ`, this definition can be found
+in `measure_theory.measure.vector_measure` and is known as `measure_theory.complex_measure`.
+
+## Main definitions
+
+* `measure_theory.complex_measure.re`: obtains a signed measure `s` from a complex measure `c`
+  such that `s i = (c i).re` for all measurable sets `i`.
+* `measure_theory.complex_measure.im`: obtains a signed measure `s` from a complex measure `c`
+  such that `s i = (c i).im` for all measurable sets `i`.
+* `measure_theory.signed_measure.to_complex_measure`: given two signed measures `s` and `t`,
+  `s.to_complex_measure t` provides a complex measure of the form `s + it`.
+* `measure_theory.complex_measure.equiv_signed_measure`: is the equivalence between the complex
+  measures and the type of the product of the signed measures with itself.
+
+# Tags
+
+Complex measure
+-/
+
+noncomputable theory
+open_locale classical measure_theory ennreal nnreal
+
+variables {α β : Type*} {m : measurable_space α}
+
+namespace measure_theory
+
+open vector_measure
+
+namespace complex_measure
+
+include m
+
+/-- The real part of a complex measure is a signed measure. -/
+@[simps]
+def re (c : complex_measure α) : signed_measure α :=
+c.map_range complex.re_lm.to_add_monoid_hom complex.continuous_re
+
+lemma re_add (c d : complex_measure α) :
+  (c + d).re = c.re + d.re :=
+rfl
```

Comment:
Any reason not to make this `simp`? Also, can you add the statement that `0.re = 0`?